grc/gui/MainWindow.py

```diff
@@ -58,18 +59,29 @@ def __init__(self, app, platform, action_handler_callback):
         log.debug("__init__()")
 
         self._platform = platform
+        self.app = app
         self.config = platform.config
 
+        # Add all "win" actions to the local
+        win_actions = filter(lambda x: x.startswith("win."), Actions.get_actions())
+        map(lambda x: self.add_action(Actions.actions[x]), win_actions)
```

Comment:
map and lambda have valid use cases.

As no list is build here, a list comprehension isn't much better here IMO.

in python3 map() is a generator. I'm not really if this even had the intended function